fhir-mapping/src/main/ml-modules/root/data-services/utils/opticQueryUtils.sjs

```diff
@@ -5,7 +5,7 @@ function applyStringParamModifier(modifier, value) {
     //do nothing to the value
   } else if("contains" === modifier) {
     value = '%' + value + '%'
-  } else if("" === modifier) {
+  } else if(modifier === null || "" === modifier) {
```

Comment:
This could be reduced to `!modifier`

Reply:
would that match anything other than null and empty string? I am always cautious when using javascripts boolean logic on anything but booleans.

Reply:
It would match `null`, `undefined`, `0`, `''`, and `false` as those are all "falsy" values.